README.md

```diff
@@ -44,11 +44,22 @@
 
 ## Getting started
 
-- Clone the repository: `git clone https://github.com/developmentseed/eoAPI.git`
-- Navigate to the project: `cd eoAPI`
-- Run services with `docker compose up`
-- Follow the [MAXAR open data demo](https://github.com/vincentsarago/MAXAR_opendata_to_pgstac) (or get inspired by the other [demos](./demo/)) to load some data into eoAPI
-- Check out the [Search Viewer](http://localhost:8081/index.html), and the API documentation ([STAC Metadata](http://localhost:8081/docs), [Raster Tiles](http://localhost:8082/docs), [Vector Tiles](http://localhost:8083/api.html))
+The easiest way to start exploring the different eoAPI services is with *Docker*. Clone this repository and start the multi-container *Docker* applications using `Compose`:
+
+```
+git clone https://github.com/developmentseed/eoAPI.git
+cd eoAPI
+docker compose up
+```
```

Comment:
This might be a personal habit but I usually see more easily code blocks than multiple lines